server/src/main/java/io/spine/server/command/model/CommandAcceptingSignature.java

```diff
@@ -79,7 +88,7 @@
         MESSAGE {
             @Override
             public boolean matches(MethodParams params) {
-                return params.is(CommandMessage.class);
+                return params.is(CommandMessage.class) && params.is(GeneratedMessageV3.class);
```

Comment:
Why `GeneratedMessageV3`? I suppose, `Message` provides a sufficient interface. And `CommandMessage` extends `Message`.